js/levels.js

```diff
@@ -739,7 +739,7 @@ var levels = [
       'fr': '<p>Jusqu\'à présent, vous aviez mis votre jardin dans une grille avec cinq colonnes, chacune ayant une largeur de 20%, et cinq lignes, chacune ayant une hauteur de 20%.</p><p>Cela a été fait avec les règles <code>grid-template-columns: 20% 20% 20% 20% 20%;</code> et <code>grid-template-rows: 20% 20% 20% 20% 20%;</code> Chaque règle comporte cinq valeurs qui créent cinq colonnes, chacune correspond à 20% de la largeur totale du jardin.</p><p>Mais vous pouvez configurer la grille comme vous le souhaitez. Donnez une nouvelle valeur à <code>grid-template-columns</code> pour arroser vos carottes. Vous devez définir la largeur de la 1ère colonne à 50%.',
       'de': '<p>Bis jetzt war dein Garten als Raster von fünf Spalten mit je 20% der Gesamtbreite sowie fünf Reihen mit je 20% der Gesamthöhe angelegt.</p><p>Dies wurde durch die Regeln <code>grid-template-columns: 20% 20% 20% 20% 20%;</code> und <code>grid-template-rows: 20% 20% 20% 20% 20%;</code> bewirkt. Jede dieser Regeln hat fünf Werte, was fünf Spalten erzeugt, jede 20% der Gesamtbreite des Gartens.</p><p>Aber du kannst das Raster so konfigurieren wie du möchtest. Gib <code>grid-template-columns</code> einen neuen Wert um deine Karotten zu wässern. Du solltest die Breite der ersten Spalte auf 50% setzen.',
-      'nl': '<p>Tot op dit moment, heb je je tuin ingesteld gehad op een grid met vijf kolommen, elke met een breedte van 20% van de volledige breedte, en vijf rijen, elke 20% van de volledige hoogte.</p><p>Dit was gedaan met de regels <code>grid-template-columns: 20% 20% 20% 20% 20%;</code> en <code>grid-template-rows: 20% 20% 20% 20% 20%;</code> Iedere regel had vijf waardes wat vijf kolommen maakt, elke ingesteld met 20% van de gehele tuin breedte.</p><p>Maar je kan het grid instellen zoals je wilt. Geef <code>grid-template-columns</code> een nieuwe waarde om je planten water te geven. Je wil dat de eerste kolom een breedte heeft van 50%.',
+      'nl': '<p>Tot nu spande je tuin zich uit over een grid met vijf kolommen, elke met een breedte van 20 % van de volledige breedte, en vijf rijen, elke 20 % van de volledige hoogte.</p><p>Dit is gedaan met de regels <code>grid-template-columns: 20% 20% 20% 20% 20%;</code> en <code>grid-template-rows: 20% 20% 20% 20% 20%;</code> Iedere regel had vijf waardes, waardoor vijf kolommen ontstonden, elke ingesteld met 20 % van de gehele tuinbreedte.</p><p>Maar je kan het grid instellen zoals je wilt. Geef <code>grid-template-columns</code> een nieuwe waarde om je planten water te geven. Je wilt dat de eerste kolom een breedte heeft van 50 %.',
```

Comment:
Why did you put a space between the number and percentages? See the `20 %`s and the `50 %` in the end. In CSS you neither put a space between the number and the unit, so this might cause confusion.

Reply:
See https://taaladvies.net/taal/advies/tekst/114/wel_of_geen_spaties_voor_en_na_leestekens_en_symbolen_algemeen/#10